src/test/Makefile

```diff
@@ -465,7 +472,14 @@ ifeq ($(shell $(CC) -v 2>&1 | grep -q "clang version" && echo "clang"),clang)
 # Please revisit versions when new clang version arrive. Supported versions: { Linux: 7 - 11; OSX: 7- 12 }
 # Travis reports CC_VERSION of 4.2.1
 CC_VERSION_MAJOR := $(firstword $(subst ., ,$(CC_VERSION)))
+
+# On Cygwin, the latest version of clang 8.0.1 reports itself as version 4.2.1 when using the `-dumpversion` option, when using `--version` the result is 8.0.1
```

Comment:
This is not a bug in Betaflight, but a bug in cygwin. Have you filed a bug with cygwin so that this can get fixed and this workaround removed?

Reply:
Update: This was a bug in clang itself that was fixed in clang 9 about 2 years ago - all versions (even ones that will fail to build our tests) prior to this report themselves as '4.2.1' - so there is no way that your check will be able to determine if a version reporting itself as '4.2.1' can be used to build the tests.

Reply:
> Excluding a test not working with clang version used by cygwin is not a solution. Rather try to update clang itself.

this is specifically labelled as a workaround, not a solution.  The first thing I did was to find a newer clang package but one does not exist yet.